proto/minder/v1/minder.proto

```diff
@@ -2590,6 +2590,9 @@ message EvaluationHistoryStatus {
     // details contains optional details about the evaluation.
     // the structure and contents are rule type specific, and are subject to change.
     string details = 2;
+
+    // created_at is the timestamp of creation of this evaluation
+    google.protobuf.Timestamp evaluated_at = 3;
```

Comment:
This field was missing from the spec.
@dmjb is this the right place?

Reply:
Yes, that makes sense